tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -184,3 +185,48 @@ def test_rh_here_objects(self, cluster):
         save_test_table_remote()
         assert "test_table" in cluster.keys()
         assert isinstance(cluster.get("test_table"), rh.Table)
+
+    @pytest.mark.level("local")
+    def test_rh_status_local(self, cluster):
+        res = subprocess.check_output(["runhouse", "status", cluster.name]).decode(
+            "utf-8"
+        )
+        assert "😈 Runhouse Daemon is running 🏃" in res
+        assert f"server_port: {cluster.server_port}" in res
+        assert f"server_connection_type: {cluster.server_connection_type}" in res
+        assert f"den_auth: {str(cluster.den_auth)}" in res
+        assert f"resource_type: {cluster.RESOURCE_TYPE.lower()}" in res
+        assert f"ips: {str(cluster.ips)}" in res
+        assert "Serving 🍦 :" in res
+
+        cluster_keys = cluster.keys()
+        for key in cluster_keys:
+            assert key in res
+
+    @pytest.mark.level("local")
+    def test_rh_status_in_cluster(self, cluster):
+        res = cluster.run(["runhouse status"])[0][1]
+        assert "😈 Runhouse Daemon is running 🏃" in res
+        assert f"server_port: {cluster.server_port}" in res
+        assert f"server_connection_type: {cluster.server_connection_type}" in res
+        assert f"den_auth: {str(cluster.den_auth)}" in res
+        assert f"resource_type: {cluster.RESOURCE_TYPE.lower()}" in res
+        assert f"ips: {str(cluster.ips)}" in res
+        assert "Serving 🍦 :" in res
+
+        cluster_keys = cluster.keys()
+        for key in cluster_keys:
+            assert key in res
+
+    @pytest.mark.level("local")
+    def test_rh_status_errors(self, docker_cluster_pk_ssh_no_auth):
```

Comment:
any reason to use a specific fixture here and not just "cluster" which will be parametrized across multiple types?